editor.planx.uk/src/@planx/components/List/Editor.tsx

```diff
@@ -92,6 +94,8 @@ function ListComponent(props: Props) {
     validateOnChange: false,
   });
 
+  const dataFieldSchema = useStore().getFlowSchema()?.nodes;
```

Comment:
`List`, `Page` & `MapAndLabel` components have their own pre-existing concepts of "schemas", so the store function is more explicitly assigned to `dataFieldSchema` in these cases